frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx

```diff
@@ -26,10 +26,10 @@ const filtersToLocalFilters = (filters: RecordingFilters): LocalRecordingFilters
         actions: [],
         events: [
             {
-                id: null,
+                id: '$pageview',
```

Comment:
because session recordings defaults to loading 21 days of data, defaulting to "All events" for the filter is _reeeaaaallllyyyy_ slow